packages/bridge/contracts/l1/L1GovernanceRelay.sol

```diff
@@ -69,7 +81,7 @@ contract L1GovernanceRelay {
       targetData
     );
 
-    IInbox(inbox).createRetryableTicket{value: msg.value}(
```

Comment:
Can this contract instead inherit from `L1CrossDomainEnabled`?
Could update the `sendTxToL2`  function there to include `l1CallValue` then it would work neatly over here.

Reply:
I thought about this too but then we need to add paramts to `sendTxToL2` which will cause stack problems again :/

Reply:
done by adding overload